### PR TITLE
Browser logging instructions moved to browser dev

### PR DESCRIPTION
**Pull request by rainemak**
Browser logging instructions moved to browser development aka working with browser page.

### CODE REVIEW

File: Reference/Core_Areas_and_APIs/Browser/README.md [hunk 1]
```diff
@@ -13,10 +13,4 @@ The web browser in Sailfish OS is a fully open-source component which has been d
...
 ## Debugging
 
-For newer versions of Gecko, the environment variable MOZ_LOG is used to enable logging for various modules. There is no option to turn all logging on, so explicit module names must be found in the source (grep for LazyLogModule).
-
-Useful module names we've used so far include:
-
-  - nsComponentManager - Component loading and unloading
-  - GMP - Gecko Media Plugin activity
-  - PlatformDecoderModule - system codecs such as ffmpeg
+Logging instruction can be found [browser development page](/Reference/Core_Areas_and_APIs/Browser/Working_with_Browser/#enabling-module-log-output)
```
Comment (llewelld) [inline Reference/Core_Areas_and_APIs/Browser/README.md:16]:
```suggestion
Logging instruction can be found on the [browser development page](/Reference/Core_Areas_and_APIs/Browser/Working_with_Browser/#enabling-module-log-output).
```

**Review by llewelld — APPROVED** [review]
I've made a small suggestion, but alongside #52 this looks good.

**Comment by rainemak** [comment]
@llewelld squashed your change and amended "Co-authored-by: David Llewellyn-Jones <david.llewellyn-jones@jolla.com>" to the commit.

**Comment by rainemak** [comment]
Rebased

**Review by llewelld — APPROVED** [review]
"Co-authored" is overstating it just a bit :D (do feel free to drop that). But, that aside, this LGTM.

**Pull request merged**